app/src/components/Scheduling/ContactReview.js

```diff
@@ -0,0 +1,150 @@
+  };
+
+  return (
+    <div className={classes.root}>
+      <Header
+        title="Contact Information"
+        description="Review your profile information below"
+        activity={`Training with ${HARD_CODED_DATA.caseWorker}`}
+      />
+
+      <FormControl>
+        <>
+          <label className={classes.letterSpacing}>First Name<span className={classes.required}>*</span>
+          </label>
+          <TextField
+            className={classes.formSpace}
+            variant="outlined"
+            fullWidth
+            name="firstName"
+            value={inputs.firstName}
+            inputProps={{
+                readOnly: true,
+                "aria-readonly": true,
+              }}
+            required
+          />
+        </>
+        <>
+          <label className={classes.letterSpacing}>
+            Last Name<span className={classes.required}>*</span>
+          </label>
+          <TextField
+            className={classes.formSpace}
+            variant="outlined"
+            fullWidth
+            name="lastName"
+            value={inputs.lastName}
+            inputProps={{
+              readOnly: true,
+              "aria-readonly": true,
+            }}
+            required
+          />
+        </>
+        <>
+          <label className={classes.letterSpacing}>Email Address<span className={classes.required}>*</span>
+          </label>
+          <div className={classes.input}>
+          <TextField
+            className={classes.formSpace}
+            type="email"
+            variant="outlined"
+            fullWidth
+            name="email"
+            value={inputs.email}
+            onChange={handleChange}
+            disabled={!editMode.email}
+            InputProps={{
+              classes: {
+                disabled: classes.disabled,
+                input: classes.inputWidth
+              },
+            }}
+            required
+          />
+          <Button
+            style={{bottom: '9rem'}}
+            color="primary"
+            className={classes.editButton}
+            onClick={() => toggleEdit({ ...editMode, email: !editMode.email})}
+          >
+            {editMode.email ? 'confirm' : 'edit' }
+          </Button>
+          </div>
+        </>
+        <>
+          <label className={classes.letterSpacing}>Phone Number (Optional)</label>
+          <div className={classes.input}>
+          <TextField
+            className={classes.formSpace}
+            type="text"
+            variant="outlined"
+            fullWidth
+            name="phoneNumber"
+            value={inputs.phoneNumber}
+            onChange={handleChange}
+            disabled={!editMode.phoneNumber}
+            InputProps={{
+              classes: {
+                disabled: classes.disabled,
+                input: classes.inputWidth
+              },
+            }}
+          />
+          <Button
+            style={{bottom: '2.7rem'}}
```

Comment:
Remove this line so we can keep all styles in one place. Same for line 105, and a few other places in different components.